### PR TITLE
MAP: Handle transparency in the map

### PR DESCRIPTION
**Pull request by jwiggins**
We now draw a background behind the map so that we can "knock out" tiles in the map when the worm digs

### CODE REVIEW

File: assets.wasmp [hunk 1]
```diff
@@ -218,6 +218,6 @@
 -- </FLAGS>
 
 -- <PALETTE>
--- 000:14181c5db2e2ff95aabea18dffcd75a7f07038b764257179be7565813c3820506dfad620f4f4f494b0c2566c86333c57
+-- 000:0000005db2e2ff95aabea18dffcd75a7f07038b764257179be7565813c3820506dfad620f4f4f494b0c2566c86333c57
```
Comment (jwiggins) [inline assets.wasmp]: Just setting the first palette entry to 0x000000

File: assets.wasmp [hunk 2]
```diff
@@ -58,133 +58,133 @@
...
 -- 008:202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020203030303030303030302020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020
--- 009:101020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020203030303030303030302020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020202020201010
```
Comment (JR-Mitchell) [inline assets.wasmp:96]: What is this change?

Comment (jwiggins) [reply assets.wasmp:96]: I added background tiles to the right edge of the map

Comment (jwiggins) [reply assets.wasmp:96]: Rightmost column of the map is just background now. We can't go there any more.

**Pull request merged**